tests/attacks/test_virtual_adversarial.py

```diff
@@ -213,20 +221,21 @@ def test_iris_k_unbounded(self):
     #     acc = np.sum(preds_adv == np.argmax(y_test, axis=1)) / y_test.shape[0]
     #     logger.info('Accuracy on Iris with VAT adversarial examples: %.2f%%', (acc * 100))
 
-    def test_iris_pt(self):
```

Comment:
Why did you comment `test_iris_pt`?

Reply:
The test_iris_pt and the test_iris_tf get the same error as the attack produces "nan" in both cases as the adversarials. So there maybe problem with the attack itself or the supportability of the attack for vector input. This needs to be investigated more.